PocketSVGTests/PocketSVGTests.swift

```diff
@@ -82,5 +82,54 @@ class PocketSVGTests: XCTestCase {
 
         XCTAssertEqual(rectanglePath.svgRepresentation, representation)
     }
-    
+
+    func testIgnoresMaskElement() {
+        let svgString = """
+            <svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px">
+                <g>
+                    <mask>
+                        <g>
+                            <rect width="100" height="100" style="fill: #FFFFFF" />
+                        </g>
+                    </mask>
+                    <rect x="20" y="20" width="60" height="60" style="fill: #FF0000"/>
+                </g>
+            </svg>
+            """
+
+        let paths = SVGBezierPath.paths(fromSVGString: svgString)
+        XCTAssertEqual(paths.count, 1)
+
+        guard let path = paths.first else {
```

Comment:
if `XCTAssertEqual(paths.count, 1)` passes, then we know that `paths.first != nil`, so this `guard let` is redundant and it's safe to do `path!.bounds` below.

Reply:
I figured that bailing after failing the assertion was friendlier than hitting a fatal error and breaking the whole test run in the case that we don't parse any paths out of the svg?

Reply:
an individual test's execution stops as soon as an assertion fails, so if we don't parse any paths out of the svg we'll duly fail the assertion on line 101 and we're guaranteed not to execute the code with the force unwrap.

Reply:
Hm. That’s not how Xcode (9.3) behaves on my computer and I can’t see `.continueAfterFailure` being set anywhere. Is that a global configuration option in Xcode?

Reply:
ah, sorry, you're absolutely right, I hadn't set `.continueAfterFailure` to `true`

I have now: https://github.com/pocketsvg/PocketSVG/commit/1c014fa085f35891e7bc187cb6c5698fe48dd2f4